doc/whats_new/v0.22.rst

```diff
@@ -85,6 +85,10 @@ Changelog
   between `n_jobs=1` and `n_jobs>1` due to the handling of the random state.
   :pr:`9288` by :user:`Bryan Yang <bryanyang0528>`.
 
+- |Fix| Fixed a bug where `elkan` algorithm in :class:`cluster.KMeans` was
+  producing Segmentation Fault on large arrays due to integer index overflow.
+  :pr:`15057` by :user:`Vladimir Korolev <balodja@gmail.com>`.
```

Comment:
:user:`Vladimir Korolev <balodja@gmail.com>`

it needs to have your github username, not your email

Reply:
Oh, I see. Changed.